apache-poi-3/src/test/java/com/baeldung/jexcel/JExcelDetectEmptyRowUnitTest.java

```diff
@@ -0,0 +1,44 @@
+package com.baeldung.jexcel;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import org.junit.Before;
+import org.junit.Test;
+
+import jxl.Cell;
+import jxl.Sheet;
+import jxl.Workbook;
+import jxl.read.biff.BiffException;
+
+
+public class JExcelDetectEmptyRowUnitTest {
```

Comment:
Needs to be formatted as per the standard - this uses tabs

Reply:
Done, please check.